<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-FAP6QC75" failures="3" tests="15" name="acj79_RegistrationFirstName.PositiveSignUpTests" time="175.268" errors="0" timestamp="2021-01-22T21:00:11 PST" skipped="0">
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.127"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.435"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="16.425"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="11.959"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="9.941"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="11.112"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.466"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="13.419">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at reusableUtilities.ReusableMethods.assertEqualStrings(ReusableMethods.java:202)
at pageObjects.SignUpPageObject.validateSignUpPositiveScenarios(SignUpPageObject.java:61)
at acj79_RegistrationFirstName.PositiveSignUpTests.validateAllPositiveScenariosForSignUpTest(PositiveSignUpTests.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- validateAllPositiveScenariosForSignUpTest -->
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="13.934"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="12.078">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at reusableUtilities.ReusableMethods.assertEqualStrings(ReusableMethods.java:202)
at pageObjects.SignUpPageObject.validateSignUpPositiveScenarios(SignUpPageObject.java:61)
at acj79_RegistrationFirstName.PositiveSignUpTests.validateAllPositiveScenariosForSignUpTest(PositiveSignUpTests.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- validateAllPositiveScenariosForSignUpTest -->
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.931"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="9.745"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.516"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="10.226"/>
  <system-out/>
  <testcase classname="acj79_RegistrationFirstName.PositiveSignUpTests" name="validateAllPositiveScenariosForSignUpTest" time="13.954">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at reusableUtilities.ReusableMethods.assertEqualStrings(ReusableMethods.java:202)
at pageObjects.SignUpPageObject.validateSignUpPositiveScenarios(SignUpPageObject.java:61)
at acj79_RegistrationFirstName.PositiveSignUpTests.validateAllPositiveScenariosForSignUpTest(PositiveSignUpTests.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- validateAllPositiveScenariosForSignUpTest -->
  <system-out/>
</testsuite> <!-- acj79_RegistrationFirstName.PositiveSignUpTests -->
